backside/README.md

```diff
@@ -0,0 +1,13 @@
+Setup:
+
+    dep init
+    
+    npm i
+
+    sudo npm i -g serverless-offline
+
+    sudo npm i -g serverless
```

Comment:
`serverless-offline` and `serverless` packages are listed in the [package.json](https://github.com/sk8mate/sk8/pull/15/files#diff-f4e58449e174919414e02180f2a1b2cae683df519883ffe1c2d519f9a6017de3) file. They will be installed locally after running `npm i` command, so there is no need to install them globally. You can safely remove this.